tests/Integration/ErrorReporting/ErrorReportingTest.php

```diff
@@ -0,0 +1,293 @@
+<?php
+
+namespace DDTrace\Tests\Integration\ErrorReporting;
+
+use DDTrace\Tests\Common\WebFrameworkTestCase;
+use DDTrace\Tests\Frameworks\Util\Request\GetSpec;
+
+if (PHP_VERSION_ID >= 80000) {
```

Comment:
For the future, you can also use phpunit native method to skip entire test classes. https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html#skipping-tests

With the phppunit approach, when you finally enable it for all versions, it is one line only to review instead of a file-long change (all lines shifting back 4 characters).

Reply:
Good point, wasn't aware of that :-)